safety/scan/render.py

```diff
@@ -337,22 +337,36 @@ def print_wait_policy_download(console: Console, closure: Tuple[Any, Dict[str, A
     Returns:
         Optional[PolicyFileModel]: The downloaded policy file model.
     """
+    # can replace with tenacity in the future
+    MAX_RETRIES = 3
+    INITIAL_BACKOFF = 1  # seconds
```

Comment:
Can we start with lower initial retry i.e `0.2` which is 200 ms and bump the retries to 5.